code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -65,3 +65,71 @@
 /obj/item/clothing/gloves/rapid/ComponentInitialize()
 	. = ..()
 	AddComponent(/datum/component/wearertargeting/punchcooldown)
+
+
+/obj/item/clothing/gloves/color/plasmaman
+	desc = "Covers up those scandalous boney hands."
+	name = "plasma envirogloves"
+	icon_state = "plasmaman"
+	item_state = "plasmaman"
+	cold_protection = HANDS
+	min_cold_protection_temperature = GLOVES_MIN_TEMP_PROTECT
+	heat_protection = HANDS
+	max_heat_protection_temperature = GLOVES_MAX_TEMP_PROTECT
+	resistance_flags = NONE
+
+/obj/item/clothing/gloves/color/plasmaman/black
+	name = "black envirogloves"
+	icon_state = "blackplasma"
+	item_state = "blackplasma"
+
+/obj/item/clothing/gloves/color/plasmaman/white
+	name = "white envirogloves"
+	icon_state = "whiteplasma"
+	item_state = "whiteplasma"
+
+/obj/item/clothing/gloves/color/plasmaman/robot
+	name = "roboticist envirogloves"
+	icon_state = "robotplasma"
+	item_state = "robotplasma"
+
+/obj/item/clothing/gloves/color/plasmaman/janny
+	name = "janitor envirogloves"
+	icon_state = "jannyplasma"
+	item_state = "jannyplasma"
+
+/obj/item/clothing/gloves/color/plasmaman/cargo
+	name = "cargo envirogloves"
+	icon_state = "cargoplasma"
+	item_state = "cargoplasma"
+
+/obj/item/clothing/gloves/color/plasmaman/engineer
+	name = "engineering envirogloves"
+	icon_state = "engieplasma"
+	item_state = "engieplasma"
+
+/obj/item/clothing/gloves/color/plasmaman/atmos
+	name = "atmos envirogloves"
+	icon_state = "atmosplasma"
+	item_state = "atmosplasma"
+
+/obj/item/clothing/gloves/color/plasmaman/explorer
+	name = "explorer envirogloves"
+	icon_state = "explorerplasma"
+	item_state = "explorerplasma"
+
+/obj/item/clothing/gloves/color/botanic_leather/plasmaman
+	name = "botany envirogloves"
+	desc = "Covers up those scandalous boney hands."
+	icon_state = "botanyplasma"
+	item_state = "botanyplasma"
+
+/obj/item/clothing/gloves/color/plasmaman/prototype
+	name = "prototype envirogloves"
+	icon_state = "protoplasma"
+	item_state = "protoplasma"
+
+/obj/item/clothing/gloves/color/plasmaman/clown
+	name = "clown envirogloves"
+	icon_state = "clownplasma"
+	item_state = "clownplasma"
```

Comment:
Your linters, hand them over.

Reply:
what is happen

Reply:
explain

Reply:
I looked at the files changed and saw the pesky no newline icon, so i thought, hey, let's review this so pire notices.

Reply:
Resolved due to commit 57a4997

Reply:
this is very good and epic I'm going to give you a big sloppy kiss :flushed:

Reply:
how come spritebot is kill